### ISSUE TITLE
Start on new and chainability

### ISSUE THREAD

**Issue by jeremychone** [nkreeger/node-simple-timer#1]
Love the simplicity, but I would love 

**Start on new:** 
```ts
const timer = new Timer(true); // would call start on create
```

**Chainability:**
~~timer.reset().start();~~ (I realized that start was setting hrEnd to null, as it should)

```ts
timer.end().milliseconds();
```

I can do the pull request if you want.

**Comment by nkreeger** [comment]
Good idea - https://github.com/nkreeger/node-simple-timer/pull/2

**Comment by jeremychone** [comment]
If 1) is not desired, what about `const timer = new Timer().start()` (kind of look odd, but will work)

**Comment by nkreeger** [comment]
Added an option bool to the constructor:

```js
const timer = new Timer(true);  // autostart
```

**Comment by jeremychone** [comment]
Ha, ok, great.

**Comment by nkreeger** [comment]
Fixed in 0.1.1: https://www.npmjs.com/package/node-simple-timer

**Comment by jeremychone** [comment]
Super!
